nemethmate_homework_9

Methmat opened the pull request.

Methmat committed:
nemethmate_homework_9

NagyIstvanGabor reviewed (changes requested):
Szia!

A models.py és a schemas.py üresek. Ránézel légyszi?

Methmat committed:
nemethmate_homework_9 update

Szia, bocsi teljesen lemaradtam ezeketről az utlsó kommentekről. Valszeg nem mentettem el a fileokat de megírtam őket újra. Köszi

NagyIstvanGabor reviewed (changes requested):
Szia!

Ezt még így nem tudom approve-olni, ha van még időd/energiád ezzel foglalkozni így a kurzus után is, akkor nézd vissza a 17.-es órát. Próbáld meg az órai anyag mentén haladva felépíteni a házi feladatot is. A logika teljes mértékben ugyanaz, csak a témakör más.
Ha elakadsz valahol, jelezd discordon. Köszi!
